eng/scripts/Get-PkgVersion.ps1

```diff
@@ -7,26 +7,36 @@ param (
     [string] $PackageName
 )
 
-. ${PSScriptRoot}/SdkVersion-Common.ps1
+$repoRoot = Resolve-Path "$PSScriptRoot/../..";
+. ${repoRoot}\eng\common\scripts\logging.ps1
```

Comment:
Lets make sure these paths correct work as expected on non-windows. If they don't we might need to reverse the slashes or maybe switch to Join-Path.